dev/customer_testing/lib/customer_test.dart

```diff
@@ -26,6 +27,14 @@ class CustomerTest {
         fetch.add(line.substring(6));
       } else if (line.startsWith('update=')) {
         update.add(Directory(line.substring(7)));
+      } else if (line.startsWith('iterations=')) {
```

Comment:
Will this now mean that this directive appearing will override the default value of 1 that gets used in flutter/flutter CI?

Reply:
no, see runner.dart (and run_tests.dart) below. It uses the lower of the two values.